src/com/oltpbenchmark/benchmarks/tpcc/procedures/Delivery.java

```diff
@@ -85,15 +88,24 @@
       "   AND C_ID = ? ");
 
   // Delivery Txn
-  private PreparedStatement delivGetOrderId = null;
-  private PreparedStatement delivDeleteNewOrder = null;
-  private PreparedStatement delivGetCustId = null;
-  private PreparedStatement delivUpdateCarrierId = null;
-  private PreparedStatement delivUpdateDeliveryDate = null;
-  private PreparedStatement delivSumOrderAmount = null;
-  private PreparedStatement delivUpdateCustBalDelivCnt = null;
-
-
+  private InstrumentedPreparedStatement delivGetOrderId = null;
+  private InstrumentedPreparedStatement delivDeleteNewOrder = null;
+  private InstrumentedPreparedStatement delivGetCustId = null;
+  private InstrumentedPreparedStatement delivUpdateCarrierId = null;
+  private InstrumentedPreparedStatement delivUpdateDeliveryDate = null;
+  private InstrumentedPreparedStatement delivSumOrderAmount = null;
+  private InstrumentedPreparedStatement delivUpdateCustBalDelivCnt = null;
+
+  public static void printLatencyStats() {
```

Comment:
I wish there was an easier way to display this. Maybe @robertsami knows a trick here?
But i guess it is fine.